packages/gestalt/src/__snapshots__/Dropdown.jsdom.test.js.snap

```diff
@@ -20,16 +20,16 @@ exports[`Dropdown renders a menu of 3 items conditionally 1`] = `
         tabindex="0"
       />
       <div
-        class="container rounding4 contents maxDimensions minDimensions"
-        style="position: absolute; left: 0px; top: 0px; visibility: visible; transform: translate(0px, 0px);"
+        class="absolute block borderBox shadow rounding4"
+        style="position: absolute; left: 0px; top: 0px; visibility: visible; outline: none; transform: translate(0px, 0px);"
         tabindex="0"
       >
         <div
           aria-label="Dropdown"
-          class="border rounding4 innerContents maxDimensions minDimensions primary"
+          class="relative maxDimensions minDimensions border primary rounding4"
           id="ex-1"
           role="menu"
-          style="max-width: 360px;"
+          style="max-width: 360px; overflow: auto;"
```

Comment:
overflow rule used to be in the `innerContents` class. I deleted class and now overflow is set in `style` attribute.